.github/workflows/run-unit-tests.yml

```diff
@@ -224,6 +235,19 @@ jobs:
         run: |
           swift test ${PACKAGE_ROOT} ${WITH_TSAN} ${TEST_FILTER} ${EXTRA_FLAGS}
 
+  ios-unit:
+    if: ${{ !(github.event.pull_request.draft || false) && inputs.ios_scheme_name != '' }}
+    runs-on: macos-latest
+    steps:
+      - name: Select appropriate Xcode version
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: 16.0
```

Comment:
I'd suggest `latest-stable` here.

Reply:
This currently points to Xcode 15 so won't work with Swift 6

Reply:
Ok turns out for some utterly bizarre reason GH are going to remove Xcode 16 from the macOS 14 runners, even though they're marked as `macOS-latest` 🤯 

I've manually set the macOS version to 15 which has Xcode 16 as latest stable

Reply:
Bizarre is definitely the word for it... Well, maybe they'll have the Sequoia runners up sooner than later for once.